content/doc/book/managing/index.adoc

```diff
@@ -13,15 +13,146 @@ endif::[]
-For an overview of content in the Jenkins User Handbook, see
-<<getting-started#,User Handbook overview>>.
+== System Configuration group
+
+Screens for configuring resources for your Jenkins instance.
+
+link:system-configuration[Configure System]::
+Configure global settings and paths for the Jenkins instance
+
+link:tools.adoc[Global Tool Configuration]::
+Configure tools, their locations, and automatic installers
+
+link:plugins.adoc[Manage Plugins]::
+Add, update, remove, disable/enable plugins
+that extend the functionality of Jenkins.
+
+link:nodes.adoc[Manage Nodes and Clouds]::
+Add, remove, control, and monitor the nodes used for the agents on which build jobs run.
+
+link:casc.adoc[Configuration as Code]::
+Configure your Jenkins instance using a human-readable YAML file rather than the UI.
+This is an optional feature that appears in this group
+only when the plugin is installed on your controller.
+
+== Security group
+
+Screens for configuring security features for your Jenkins instance.
+See link:/doc/book/security/[Securing Jenkins] for general information
+about managing Jenkins security.
+
+link:system-configuration.adoc[Configure Global Security]::
+Set configuration parameters that secure your Jenkins instance.
+
+link:/doc/book/using/using-credentials/#adding-new-global-credentials[Manage Credentials]::
+Configure the credentials that provide secure access
+to third-party sites and applications that interact with Jenkins.
+
+*Configure Credential Providers*::
+Configure credential providers and types
+
+link:users.adoc[Manage Users]::
+Manage users defined in the Jenkins user database.
+This is not used if you use a different security realm such as LDAP or AD.
+
+== Status Information group
+
+link:system-info.adoc[System Information]::
+Displays information about the Jenkins environment.
+
+link:/doc/book/system-administration/viewing-logs/[System Log]::
+Jenkins log that contains all `java.uil.logging` output related to Jenkins.
+
+*Load Statistics*::
+Displays information about resource utilization on you Jenkins instance.
+
+*Disk usage*::
+Simple disk usage estimation
+
+link:about-jenkins.adoc[About Jenkins]::
+Provides version and license information for your Jenkins instance.
+
+== Troubleshooting group
+
+*Manage Old Data*::
+Remove configuraion information related to plugins that have been removed from the instance.
+
+== Tools and Actions group
+
+Screens for common management tasks
+and management tools that enable you to do administrative tasks without using the UI.
+
+*Reload Configuration from Disk*::
+Discard all data that is loaded in memory and reload everything from the file system.
+This is useful when you modify configuration files directly on disk.
+
+link:cli-adoc[Jenkins CLI]::
+How to use the Jenkins CLI from a shell or script.
+
+link:/doc/book/managing/script-console/[Script Console]::
+Execute an Apache Groovy script for administration, troubleshooting, and diagnostics.
+
+*Prepare for Shutdown*::
```

Comment:
The definition term is already made bold by asciidoc, don't need to bold it again
```suggestion
Prepare for Shutdown::
```